### DISCUSSION TITLE
[Question] How to save progress in MAS when resetting your PC?

### DISCUSSION THREAD

**Discussion by SpaceHX** [Monika-After-Story/MonikaModDev#8578] [QA - tech support + general] [answered]
I have a question. On Windows 10 and 11, There is a feature where you can Reset Your PC. This Repairs, Uninstalls and Reinstalls either Windows 11 or 10. This also Uninstalls ALL applications on your system. But your files will be fine and will be saved. So if you were to reset your PC and just leave MAS in your files, Would the progress be saved? or would it be lost?

**Comment by Iceninja64** [comment, marked as answer]
as long as your persistent files are backed up/saved then no harm will come to your ddlc files regardless of the restart. 
but your files should be fine either way because your computer is not changing them but just saving them

**Reply by SpaceHX** [reply]
I still have a concern. It says that when you reset your PC, the process will delete and uninstall all applications. Does that include MAS? Or will it be saved since DDLC.exe is in your downloads folder?

**Reply by Iceninja64** [reply]
It shouldn't touch your DDLC but, like I said if you don't feel comfortable in trusting your computer you can always back up your persistent files to play it safe
